### DISCUSSION TITLE
How to update footer ?

### DISCUSSION THREAD

**Discussion by TNNSLAB** [adityatelange/hugo-PaperMod#883] [Q&A]
I failing to understand how I can update the footer ? 
Would appreciate some help here. 
Also how would you push the changes since the Theme is a submodule ? 
Thank you.

**Comment by mrashco** [comment]
Hi there,

I found that editing the `themes/PaperMod/layouts/partials/footer.html` works locally. But I can't get it to deploy as it's a submodule, not sure what to do here...

**Reply by mrashco** [reply]
Finally, so instead of using `git clone <PaperMod URL>.git`, I manually downloaded the `.zip` file, extracted it in the `themes/` directory and now editing the footer works. So yeah, submodule isn't ideal imo.

**Reply by mysubcult** [reply]
> Finally, so instead of using `git clone <PaperMod URL>.git`, I manually downloaded the `.zip` file, extracted it in the `themes/` directory and now editing the footer works. So yeah, submodule isn't ideal imo.

hi, you copy all "themes" folder?

**Reply by chrisrummel** [reply]
You are not supposed to edit the files in the themes-dir, but instead copy all the files that you want to change to your layouts-dir.

Example: `themes/PaperMod/layouts/partials/footer.html` to `layouts/partials/footer.html`

That way you can update the theme without overwriting your changes.
